packages/eslint-config-algolia/typescript.js

```diff
@@ -9,9 +9,5 @@ module.exports = {
       jsx: true,
     },
   },
-  extends: [
-    './base.js',
-    './rules/typescript.js',
-    'prettier/@typescript-eslint',
```

Comment:
the prettier line is gone now for typescript, have you tested if it's not needed anymore?

Reply:
it's not unneeded, it's not working anymore with the new prettier plugin.